Backend/routes/routes.py

```diff
@@ -191,3 +192,41 @@ async def update_child(id: str, child: Child):
 async def delete_child(id: str):
     ChildCollection.find_one_and_delete({"_id": ObjectId(id)})
     return {"status": "ok", "data": []}
+
+
+@child_malnutrition.post("/childMalnutrion_Add")
+async def child_malnutrition_add(child: ChildMalnutrition, child_id: str):
+    if not ChildCollection.find_one({"_id": ObjectId(child_id)}):
+        raise HTTPException(status_code=404, detail="Child not found")
+    _id = ChildMalnutritionCollection.insert_one(dict(child))
+    added_malnutrition_child = child_malnutrition_list_serializer(ChildMalnutritionCollection.find(
+        {"_id": _id.inserted_id}))
+    return {"status": "ok", "data": added_malnutrition_child}
+
+
+@child_malnutrition.get("/get_child_Malnutrition")
```

Comment:
We can rename this to get_child_malnutritions also the function name to plural as we return more than one